### PR TITLE
ffmpeg: added trim example plus augmented precision to seconds

### PR DESCRIPTION
**Pull request by FraYoshi**
Added trim line and seconds fraction precision to the line relative to frame extraction.
----
- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/ffmpeg.md [hunk 1]
```diff
@@ -16,7 +16,11 @@
 
 - Quickly extract a single frame from a video at time mm:ss and save it as a 128x128 resolution image:
 
-`ffmpeg -ss {{mm:ss}} -i {{video.mp4}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
+`ffmpeg -ss {{mm:ss.S}} -i {{video.mp4}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
+
+- Trim a video from a given time (ss) to (t) seconds from that time or just omit -t to trim everything before (ss) seconds:
```
Comment (agnivade) [inline pages/common/ffmpeg.md]: The "given time" should be in "mm:ss.S"

**Comment by FraYoshi** [comment]
Yes, the augmented precision to seconds is an English error of mine... I'm actually referring to give decimal precision to the cuts.. ^^"

File: pages/common/ffmpeg.md (hunk 1 above)
Comment (FraYoshi) [reply pages/common/ffmpeg.md]: Like....
"Trim a video from "mm:ss.S" (ss) to (t) seconds [...]" ? .-.

**Comment by agnivade** [comment]
I am wondering whether it is even necessary to mention timestamps with finer precision than a second. It is not necessary for the most common cases, and requires extra description to explain what the ".S" stands for. Not worth the extra effort IMO.

**Comment by FraYoshi** [comment]
Yes, probably better a less precise example but more clear ^^"

File: pages/common/ffmpeg.md [hunk 2]
```diff
@@ -18,6 +18,10 @@
 
 `ffmpeg -ss {{mm:ss}} -i {{video.mp4}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
 
+- Trim a video from a given time (ss) to (t) seconds from that time. Just omit the -t flag to only trim everything before (ss) seconds:
```
Comment (agnivade) [inline pages/common/ffmpeg.md]: How about "Trim a video from a given start time mm:ss to a duration of mm2:ss2 from that time. (Omit the -t flag to trim till the end):" ? And change the example duration to `mm2:ss2`

Comment (FraYoshi) [reply pages/common/ffmpeg.md]: Yes, It's more fluent! Thanks for the suggestion! :)

**Comment by agnivade** [comment]
Thank you for being patient with the review process. Appreciate it :)

**Comment by agnivade** [comment]
I forgot to ping @hvindin. Sorry it got merged already. Do you have anything to add in this PR ?

**Comment by agnivade** [comment]
I forgot to ping @hvindin. Sorry it got merged already. Do you have anything to add in this PR ?

**Comment by hvindin** [comment]
No harm done, LGTM :+1:

**Pull request merged**